### DISCUSSION TITLE
show errors from tsc?

### DISCUSSION THREAD

**Discussion by laoshaw** [typescript-eslint/typescript-eslint#8638] [Technical Discussions]
### Body

Body of the technical discussion...
on vscode if I have a wrong usage of type use(e.g. {name:string; age: number} but I use it as {string,string} in the code, vscode will show errors while typescript-eslint will show nothing until I run tsc on the code, is it possible for typescript-eslint to do some tsc sanity check like what vscode does right inside vim, so I don't need to find out it later?

### Additional Info

Any additional info...

### Before you submit your discussion, please confirm the following. If any of these required steps are not taken, we may not be able to review your RFC. Help us to help you!

- [X] I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.
- [X] I have [searched for related discussions](https://github.com/typescript-eslint/typescript-eslint/discussions) and [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues) and found none that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

**Comment by bradzacher** [comment]
We are designed to assume that your code has no syntax or type errors. We are designed to build on top of and augment TS's checks.

If you want to see TS errors - you can use `tsc` and typescript's IDE integration.

Additionally as per our contributing guide for discussions we do not use discussions for questions and enhancement requests.

https://typescript-eslint.io/contributing/discussions
